README.md

```diff
@@ -55,4 +55,21 @@ steps:
       version: v0.56.1
       cache: true
       path: "./bins"
+```
+
+## Install Trivy with non-default token
+There are cases when `github.token` (default value for `actions/checkout`) contains an invalid token for `http://github.com`.
+One of example for this when using GitHub Enterprise Server (GHES).
+See more info in https://github.com/aquasecurity/setup-trivy/issues/10
+
+To properly install Trivy, you need to populate `token` from a secret or another step (e.g. from https://github.com/actions/create-github-app-token)
+
+```yaml
+steps:
+  - name: Install Trivy
+    uses: aquasecurity/setup-trivy@v0.2.0
+    with:
+      version: v0.56.1
```

Comment:
```suggestion
      version: v0.56.2
```

Reply:
Updated in 34d9f0ee0ca364ca9f822843050fb1013babdb58